WordPress/src/androidTest/java/org/wordpress/android/util/EncryptionUtilsTest.kt

```diff
@@ -0,0 +1,91 @@
+package org.wordpress.android.util
+
+import com.goterl.lazycode.lazysodium.utils.KeyPair
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+import org.wordpress.android.util.encryptedlogging.EncryptedLogReader
+import org.wordpress.android.util.encryptedlogging.EncryptedLogWriter
+import org.wordpress.android.util.encryptedlogging.EncryptionUtils
+import java.io.File
+import java.util.UUID
+
+class EncryptionUtilsTest {
+    private lateinit var keypair: KeyPair
+
+    @Before
+    fun setup() {
+        keypair = EncryptionUtils.sodium().cryptoBoxKeypair()
+    }
+
+    @Test
+    @Throws
+    fun testThatEncryptedLogsMatchV1FileFormat() {
+        val testLogString = UUID.randomUUID().toString()
+        val log = logWithContent(testLogString)
+
+        val json = JSONObject(log.readText())
+        assertEquals(
```

Comment:
These are all spread out because they won't fit on one line and the linter didn't like that. 

IMHO this is still quite readable, but LMK if not!